src/core/lpLexer.ml

```diff
@@ -30,10 +30,7 @@ type token =
   | INDUCTIVE
   | INFIX
   | INJECTIVE
-  (*| LEFT replaced by ASSOC(Pratter.Left) *)
```

Comment:
@gabrielhdt Thanks to put again these comments both in the parser and the lexer. It helps check that no keyword is missing or added.

Reply:
No need finally.